tests/models/vivit/test_modeling_vivit.py

```diff
@@ -345,6 +345,6 @@ def test_inference_for_video_classification(self):
         self.assertEqual(outputs.logits.shape, expected_shape)
 
         # taken from original model
-        expected_slice = torch.tensor([-1.0543, 2.0764, -0.2104, 0.4439, -0.9658]).to(torch_device)
+        expected_slice = torch.tensor([-0.9498,  2.7971, -1.4049,  0.1024, -1.8353]).to(torch_device)
```

Comment:
Integration test values didn't need to be updated for EfficientNet as the model used [doesn't offset the values when rescaling by default](https://github.com/huggingface/transformers/blob/dd9d45b6ecc0861847e21d461187711331a56138/src/transformers/models/efficientnet/image_processing_efficientnet.py#L92)